### PR TITLE
CLI, PyPI package

### PR DESCRIPTION
**Pull request by patrick91**
I think it would be nice to install this via pip (or run with pipx),
so I've changed the whole project to use pdm for dependency and publishing.

I'd like to also update the cli, but maybe we can do that in another PR.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[project]
+name = "insanely-fast-whisper"
+version = "0.0.1"
+description = "A very fast whisper CLI"
+authors = [
+    { name = "VB", email = "vaibhav@europython.eu" },
```
Comment (patrick91) [inline pyproject.toml:6]: should probably use another email :)

also, is it ok to add myself here?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+readme = "README.md"
+license = { text = "MIT" }
+
+
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+[project.scripts]
+insanely-fast-whisper = "insanely_fast_whisper.cli:main"
```
Comment (patrick91) [inline pyproject.toml:25]: this will be the name of the cli `insanely-fast-whisper`, we can change :)

File: pyproject.toml (hunk 1 above)
Comment (Vaibhavs10) [reply pyproject.toml:6]: yes! please feel free to add yourself here! <3

File: pyproject.toml (hunk 2 above)
Comment (Vaibhavs10) [reply pyproject.toml:25]: I think it sounds good as is. WDYT?

File: pyproject.toml (hunk 1 above)
Comment (Vaibhavs10) [inline pyproject.toml:6]: Maybe we use reachvaibhavs10@gmail.com here.

**Review by Vaibhavs10 — APPROVED** [review]
Looks good to me to merge. Once we merge this, I'll make README changes to make this repo more about the CLI than the benchmarks :)

**Pull request merged**